searx/engines/prowlarr.py

```diff
@@ -0,0 +1,79 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+"""
+
+ Prowlarr search (Files)
+
+"""
+
+from json import loads
+from urllib.parse import urlencode
+from searx.exceptions import SearxEngineAPIException
+
+categories = ['files']
+
+paging = False
+api_key = 'None'
```

Comment:
Why not set it to empty string?

Reply:
@kvch 
ok, addressed your comments..

i wasn't quite sure what the correct approach for the "default values" are - every engine seems a bit different there.. but i changed it to empty strings now..